### PR TITLE
fix(DB/quest): Improve The Summoning Chamber

### PR DESCRIPTION
**Pull request by grimgravy**
## Changes Proposed:
- Codition: need to kill Skartax with a distance of 30
-  **_obs: the item cannot be used in everything, as this will result in multiple reappearances_**

## Issues Addressed:

- Closes

## SOURCE:

[Summoner Skartax](https://www.wowhead.com/wotlk/npc=21207/summoner-skartax#comments)
[wowhead ally](https://www.wowhead.com/tbc/quest=10585/the-summoning-chamber)
[wowhead horde](https://www.wowhead.com/tbc/quest=10602/the-summoning-chamber)

quest

## Tests Performed:

- 
- 

## How to Test the Changes:

1. add quest 10585
2. collect the item Elemental Displacer

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
@azerothcore/developers @azerothcore/testers can anyone please check this?

**Review by Nefertumm — APPROVED** [review]

**Comment by GastonCK** [comment]
Tested
Looting elemental displacer
WoWScrnShot_063023_131637

Cannot use it if Skartax is alive
WoWScrnShot_063023_133159

You may summon Infernal Oversaul multiple times
WoWScrnShot_063023_133314

Infernal Oversaul despawn after 5 minutes
WoWScrnShot_063023_134638

Quest completed
WoWScrnShot_063023_132618

**Comment by grimgravy** [comment]
The condition made by sql is not working as it should. I think the script would work better

**Comment by Helias** [comment]
Sorry @grimgravy I see that in the end you are using SAI, the PR is ready? I think it needs also another test, if so, would be better to put the label `waiting to be tested`

**Comment by grimgravy** [comment]
@Helias, this last SAI I added solves the problem of respawning every time you use the item. I was having to find a better way to solve problem, but succeeded. That's why I made the change.

**Comment by Helias** [comment]
@GastonCK could you test it again please? 🙏

**Comment by GastonCK** [comment]
Tested, overall it's the same as the previous test but also tried to summon multiple elementals.

First try:
Summon one, reset, summon another one and the first one will despawn. (in the second image you can see the first one despawning and the second one coming to attack me)
WoWScrnShot_071523_115735
WoWScrnShot_071523_121129


Second try:
Here I brought the elemental far from his spawnpoint and positioned myself in the maximum range possible to cast the elemental displacer. This time the first elemental did not despawn and I was able to summon as many elementals as I wanted, it seems that they have a range for the despawn to work and the elemental displacer's range is larger than the despawn range.
WoWScrnShot_071523_121743

**Comment by Helias** [comment]
Thanks for your test and accurate report!

`it seems that they have a range for the despawn to work and the elemental displacer's range is larger than the despawn range.`

We should increase the despawn range than, is it possible @grimgravy ?

File: data/sql/updates/pending_db_world/rev_1687180111436507500.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+(17, 0, 37285, 0, 0, 29, 0, 21207, 30, 1, 0, 0, 0, '', 'Disrupt the summoning of infernal souls within the summoning chamber of the Deathforge.');
+
+-- creature evade despawn in 5 min
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 21735) AND (`source_type` = 0) AND (`id` IN (3, 4));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(21735, 0, 3, 0, 7, 0, 100, 0, 0, 0, 0, 0, 0, 41, 300000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Infernal Oversoul - On Evade - Despawn In 300000 ms'),
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1687180111436507500.sql:9]: Use UPDATE_OOC for this one. Although it'd be better to change the spawn method of the NPC in the first place

Comment (grimgravy) [reply data/sql/updates/pending_db_world/rev_1687180111436507500.sql:9]: What would be your method?

**Comment by elthehablo** [comment]
So uhhh any updates?

File: data/sql/updates/pending_db_world/rev_1687180111436507500.sql (hunk 1 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1687180111436507500.sql:9]: @Gultask @grimgravy status

Comment (grimgravy) [reply data/sql/updates/pending_db_world/rev_1687180111436507500.sql:9]: @Kitzunu, it's not me

**Comment by Kitzunu** [comment]
Conflict

**Pull request merged**